src/main/java/server/EchoServerWrappers.java

```diff
@@ -0,0 +1,58 @@
+package server;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
```

Comment:
Subject to what others say - I'm ok with this as-is for this PR, but I would like to see the following in a future refactoring PR:  Typically, a wrapper class "wraps" around one other class, or at least wraps "one thing" (like establishing your input stream).  And each wrapper has its own interface (like, each has an interface file).  This wrapper class wraps around several classes -- ServerSocket, InputStreams, OutputStreams, etc. And there is just one interface file for all of it.  The reason you want one wrapper to wrap around one class or "one thing" is because let's say you want to expand the functionality in several places.  If you have wrapped things separately, that results in small changes to just those wrappers.  If you wrap things all in one bundle like this, this file will quickly enlarge significantly and eventually get unwieldy.  Happy to talk in person about all this.

Reply:
will refactor, in a future story. Have created new classes but can break down those even further at a future date